libs/features/webapp-prompt/webapp-prompt.css

```diff
@@ -133,6 +133,12 @@
     content: '✕';
   }
 
+  .appPrompt-close:focus {
+    background-color: #e9e9e9;
+    border-radius: 50%;
+    border: 3px solid white;
```

Comment:
This would make the border take the background color of the focused element, which is grey. Following the figma, it should be white. Am I missing something ? 🤔

Reply:
I'm not entirely sure the white border is needed in the first place, to me it looks like a spec omission, as we don't have that sort of styling for any other elements. If someone from Studio requested it specifically, let's at least turn its color and the prompt background color into a variable so we can easily update it in the future.

Reply:
Done 👍 I introduced the css variables